src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -132,7 +128,11 @@ public void exec(final URI home) throws IOException {
      * BkBasic produces headers with addresses without slashes.
      *
      * @throws IOException If some problem inside
+     * @todo #519:30min In org.takes.rq.RqHeaders.Base#map is skipping the
```

Comment:
@sebing How is this related to the current issue? Seems like a new bug to me, let's avoid ignoring unrelated tests if we introduce new bugs, can't we fix it here?